src/pages/Team/Team.js

```diff
@@ -131,13 +143,16 @@ class Team extends Component {
   };
 
   handleCloseYesNoDialog = (deleteMember) => {
-    if (this.state.selectedMember.pending === true)
-      this.handleYesNoDialogPendingMember();
-    else if (deleteMember) this.handleYesNoDialogTeamMember();
-    else
-      this.setState({
-        show: false,
-      });
+    if (deleteMember) {
+      this.deleteNestUser(this.state.selectedMember);
+    }
+    // if (this.state.selectedMember.pending === true)
+    //   this.handleYesNoDialogPendingMember();
```

Comment:
I think we can delete this and we can also delete the functions handleYesNoDialogPendingMember as well as handleYesNoDialogTeamMember if it is not used anywhere else just to clean up our code a bit